src-cpp/rdkafkacpp.h

```diff
@@ -56,27 +56,26 @@
 #include <sys/types.h>
 
 #ifdef _WIN32
-#ifndef ssize_t
-#ifndef _BASETSD_H_
-#include <basetsd.h>
-#endif
-#ifndef _SSIZE_T_DEFINED
-#define _SSIZE_T_DEFINED
+#  ifndef ssize_t
+#    ifndef _BASETSD_H_
+#      include <basetsd.h>
+#    endif
+#    ifndef _SSIZE_T_DEFINED
+#      define _SSIZE_T_DEFINED
 typedef SSIZE_T ssize_t;
+#    endif
+#  endif
+#  ifndef LIBRDKAFKA_STATICLIB
+#    ifdef LIBRDKAFKACPP_EXPORTS
+#      define RDCPP_EXPORT __declspec(dllexport)
```

Comment:
Why the change of define name?

Reply:
When building `rdkafkac++`, the C APIs in `rdkafka` (i.e. called by `rdkafkac++`) ought to be flagged as `dllimport`, rather than `dllexport`. If you use the same #define for both DLLs, then you cannot correctly switch them this way at build time.

Reply:
Okay, that makes sense, but call it RD_CPP_EXPORT.